code/modules/projectiles/guns/projectile/revolver/lemant.dm

```diff
@@ -1,230 +1,226 @@
-	var/datum/firemode/cur_mode = firemodes[sel_mode]
-
-	if(cur_mode.settings["use_launcher"])
-		shotgun.Fire(target, user, params, pointblank, reflex)
-		if(!shotgun.chambered)
-			switch_firemodes() //switch back automatically
-	else
-		..()
-
-/obj/item/gun/projectile/revolver/lemant/examine(mob/user)
-	..()
-	if(shotgun.loaded.len)
-		to_chat(user, "\The [shotgun] has \a [shotgun.chambered] loaded.")
-	else
-		to_chat(user, "\The [shotgun] is empty.")
-
-/obj/item/gun/projectile/revolver/deacon
-	name = "\"Deacon\" heavy revolver"
-	desc = "An anomalous weapon created by an unknown person (or group?), their work marked by a blue cross, these weapons are known to vanish and reappear when left alone. \
-	Some pilgrims find what they are looking for..."
-	icon = 'icons/obj/guns/projectile/lemant_blue.dmi'
-	icon_state = "lemant_blue"
-	item_state = "lemant_blue"
-	fire_sound = 'sound/weapons/guns/fire/12mm_revolver.ogg'
-	caliber = CAL_50
-	drawChargeMeter = FALSE
-	origin_tech = list(TECH_COMBAT = 3, TECH_MATERIAL = 3)
-	max_shells = 9
-	matter = list(MATERIAL_PLASTEEL = 15, MATERIAL_PLASTIC = 8)
-	price_tag = 700
-	gun_parts = null
-	damage_multiplier = 1.3
-	penetration_multiplier = 1.2
-	init_recoil = HANDGUN_RECOIL(1.1)
-	gun_tags = list(GUN_PROJECTILE, GUN_INTERNAL_MAG, GUN_REVOLVER, GUN_CALIBRE_12MM)
-	var/obj/item/gun/projectile/underslung/launcher
-	init_firemodes = list(
-		SEMI_AUTO_NODELAY,
-		list(mode_name="fire grenades", mode_desc="Shoot the underbarrel grenade shell",  burst=null, fire_delay=null, move_delay=null,  icon="grenade", use_launcher=1)
-		)
-	serial_type = "BlueCross"
-
-/obj/item/gun/projectile/revolver/deacon/Initialize()
-	. = ..()
-	launcher = new(src)
-
-/obj/item/gun/projectile/revolver/deacon/attackby(obj/item/I, mob/user)
-	if((istype(I, /obj/item/ammo_casing/grenade)))
-		launcher.load_ammo(I, user)
-	else
-		..()
-
-/obj/item/gun/projectile/revolver/deacon/attack_hand(mob/user)
-	var/datum/firemode/cur_mode = firemodes[sel_mode]
-
-	if(user.get_inactive_hand() == src && cur_mode.settings["use_launcher"])
-		launcher.unload_ammo(user)
-	else
-		..()
-
-/obj/item/gun/projectile/revolver/deacon/Fire(atom/target, mob/living/user, params, pointblank=0, reflex=0)
-	var/datum/firemode/cur_mode = firemodes[sel_mode]
-
-	if(cur_mode.settings["use_launcher"])
-		launcher.Fire(target, user, params, pointblank, reflex)
-		if(!launcher.chambered)
-			switch_firemodes() //switch back automatically
-	else
-		..()
-
-/obj/item/gun/projectile/revolver/deacon/examine(mob/user)
-	..()
-	if(launcher.loaded.len)
-		to_chat(user, "\The [launcher] has \a [launcher.chambered] loaded.")
-	else
-		to_chat(user, "\The [launcher] is empty.")
+/obj/item/gun/projectile/revolver/lemant
+	name = "\"Pilgrim\" magnum revolver"
+	desc = "An extremely outdated single-action revolver with a single-shot underbarrel shotgun. Chambered in 10mm, takes a single 20mm shell."

```

Comment:
"Chambered in 10mm and can hold a single 20mm shell" instead? Maybe also mention that it can't take speedloaders.